### ISSUE TITLE
Avoid checking links which are not URL

### ISSUE THREAD

**Issue by newptcai** [tlienart/Franklin.jl#789]
I have links like `mailto:aaa@bbb.com` and `tel:+111111` on my website. `verify_links()` report they are wrong. This should not happen. Links other than URL should simply be ignored.

**Comment by tlienart** [comment]
Yes the `verify_links` is pretty simple at the moment and is more meant as a quick check but I agree with what you report.

Note that for "serious" link checking, once your site is deployed, you'll typically want to use something like https://github.com/stevenvachon/broken-link-checker#readme
